### PR TITLE
logger implementation

### PR DESCRIPTION
**Pull request by Karthick9917**

### CODE REVIEW

File: com/ideas2it/employeeManagementSystem/dao/EmployeeDao.java [hunk 1]
```diff
@@ -287,6 +290,7 @@ public boolean updateAddress(Employee employee) throws EmsException {
             updated = insertStatement.executeBatch().length;
             insertStatement.close();
         } catch (SQLException sqlException) {
```
Comment (ponnuvelgandhi) [inline com/ideas2it/employeeManagementSystem/dao/EmployeeDao.java:303]: This log should not be used for further debug. Add actual exception

File: com/ideas2it/employeeManagementSystem/main/EmployeeManagementSystem.java [hunk 2]
```diff
@@ -11,12 +14,15 @@
...
 public class EmployeeManagementSystem {
 
+    private static Logger logger = LogManager.getLogger(EmployeeManagementSystem.class.getName());
+
     /**
      * main Method used to call the view class
      * for create, display , search, delete
      * and update employee details operations.
      */
     public static void main(String[] args) {
```
Comment (ponnuvelgandhi) [inline com/ideas2it/employeeManagementSystem/main/EmployeeManagementSystem.java]: same here

File: com/ideas2it/employeeManagementSystem/util/ConnectionUtil.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package com.ideas2it.employeeManagementSystem.util;
```
Comment (ponnuvelgandhi) [inline com/ideas2it/employeeManagementSystem/util/ConnectionUtil.java:1]: Why this class show as a new file?

File: com/ideas2it/employeeManagementSystem/util/ValidationUtil.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
+package com.ideas2it.employeeManagementSystem.util;
```
Comment (ponnuvelgandhi) [inline com/ideas2it/employeeManagementSystem/util/ValidationUtil.java:1]: Why this also?

**Review by ponnuvelgandhi — CHANGES_REQUESTED** [review]
Improve the logger information

**Review by aruni2isankar — APPROVED** [review]

**Pull request merged**